<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teaCtrl</class>
 <widget class="QMainWindow" name="teaCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教职工 - 大学生综合素质测评系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCommandLinkButton" name="testBtn">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>测评信息管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="infoBtn">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>基本信息管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="pwBtn">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>密码管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="exitBtn">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>16</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>退出系统</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正等线</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;方正等线&quot;;</string>
    </property>
    <property name="text">
     <string>您已经登录为：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正等线</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;方正等线&quot;;</string>
    </property>
    <property name="text">
     <string>您的登录名为：</string>
    </property>
   </widget>
   <widget class="QLabel" name="idLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正等线</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;方正等线&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正等线</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;方正等线&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>22</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(154, 29, 22);
font: 75 22pt &quot;华文行楷&quot;;</string>
    </property>
    <property name="text">
     <string>大学生综合素质测评系统</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="StudentTest.qrc">:/new/pics/images/sgu.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_System">
    <property name="title">
     <string>系统</string>
    </property>
    <addaction name="action_Log_out"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_System"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Log_out">
   <property name="text">
    <string>&amp;Log out</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="StudentTest.qrc"/>
 </resources>
 <connections/>
</ui>
